src/mmw/mmw/settings/layer_settings.py

```diff
@@ -148,6 +148,22 @@
             'legend_units_label': 'Slope (%)',
             'legend_unit_breaks': [0, 1, 2, 4, 6, 10, 15, 20, 30, 50, 951],
         },
+        {
+            'display': 'Elevation',
+            'code': 'elevation',
+            'css_class_prefix': 'elevation',
+            'short_display': 'Elevation',
+            'helptext': '',
+            'url': 'https://{s}.tiles.azavea.com/ned-nhdplus-30m/{z}/{x}/{y}.png',
+            'maxNativeZoom': 13,
+            'maxZoom': 18,
+            'opacity': 0.710,
+            'has_opacity_slider': True,
+            'use_color_ramp': True,
+            'color_ramp_id': 'elevation-legend',
+            'legend_units_label': 'Elevation (m)',
+            'legend_unit_breaks': [-86, 0, 20, 100, 250, 750, 2000, 4413],
```

Comment:
This is a stylistic thing, but I think the legend will look better with more regular numbers. Since the numbers are fuzzy on the gradient line, can we make the lowest number `-100` and the highest number `4500`? We can also go with `-80` and `4400` for a compromise between accuracy and readability.

Reply:
I have no preference on this, any thoughts, @ajrobbins? I think if we round the min/max here, we  should also round them on slope percent
<img width="382" alt="screen shot 2018-03-05 at 10 21 54 am" src="https://user-images.githubusercontent.com/7633670/36982989-3bf714b8-205f-11e8-8407-614958a61992.png">

Reply:
Arianna and I chatted in the kitchen, and we like being able to see the min and max on the scale; going to keep this is as for now.